podkrepi-bg/frontend
# Remove Discord widget bot

Closes #1251

## Motivation and context

We remove the widget bot and the `/chat` page as they are not actively used
## Testing

### Steps to test

1. Go to `/chat`
2. See 404 page

### Affected urls

- `/chat`

### New or updated dependencies:


Removed dependency to https://www.npmjs.com/package/@widgetbot/react-embed

Dependency name|Previous version|Updated version|Details
---|---|---|---
`@widgetbot/react-embed`|`^1.4.0`|`-`|
